catkin_home/src/navigation/robot_description/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 2.8.3)
+
+project(robot_description)
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  rospy
+)
+
+catkin_package()
+
+find_package(roslaunch)
+
+foreach(dir config launch meshes urdf)
+	install(DIRECTORY ${dir}/
+		DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/${dir})
+endforeach(dir)

```

Comment:
Interesting, no idea what is this about.

Reply:
In [this ROS page](http://wiki.ros.org/catkin/CMakeLists.txt#Optional_Step:_Specifying_Installable_Targets) it says that this is optional, and it's made only if you want to " install targets to the system so that they can be used by others or to a local folder to test a system-level installation."